MySQL connection

LucasWalters opened the pull request:
Run this in MySQL to make it work:

create database molveno; -- Creates the database
create user 'root'@'%' identified by 'admin'; -- Creates the user
grant all on molveno.* to 'root'@'%'; -- Gives all the privileges to the new user on the newly created database

GaryMikoen committed:
Added guest dashboard file

Commit:
Merge branch 'hotel' into hotel-feature/guest-dashboard

Mills315 committed:
Guest Boom Room in principe klaar.
Laatste tweaks et cetera.

Mills315 committed:
Guest Boom Room klaar voor review

Mills315 committed:
Merge branch 'hotel-feature/guest-book-room' into hotel

GaryMikoen committed:
added button to dashboard

GaryMikoen committed:
Merge branch 'hotel' into hotel-feature/guest-dashboard

Commit:
Added popup for receptionist guest data

Commit:
Merge branch 'hotel' into hotel-feature/popup-room-info-screen

Commit:
Changed header text popup

Mills315 committed:
Eerste opzet Guest Reservation Room Options Screen
html gemaakt.
nog wat aanpassingen aan oude html
Bi-Winning
Tiger Blood

Mills315 committed:
Dit is een commit message.

Mills315 committed:
laatste commit/push maandag 3/12/18

Mills315 committed:
Merge branch 'hotel' into hotel-feature/guest-reservation-room-options-screen

Commit:
More data for popup

Commit:
Merge branch 'master' into hotel

Mills315 committed:
nu echt de laatste commit/push maandag 3/12/18

Mills315 committed:
nu echt de laatste commit/push maandag 3/12/18

GaryMikoen committed:
created 6 dashboard buttons for the guesthub

Mills315 committed:
nu echt de laatste commit/push maandag 3/12/18

GaryMikoen committed:
Merge branch 'hotel' of https://github.com/mrjink/molveno into hotel

GaryMikoen committed:
Merge branch 'hotel' into hotel-feature/guest-dashboard

Commit:
Changed button style dashboard

Commit:
Merge pull request #22 from mrjink/hotel-feature/tijdelijkebranchJustin

Changed button style dashboard

Mills315 committed:
ochtend 4/12/18

Mills315 committed:
ochtend 4/12/18

Mills315 committed:
Merge branch 'hotel' into hotel-feature/guest-reservation-room-options-screen

GaryMikoen committed:
Added link in guesthub to invoice screen

GaryMikoen committed:
Merge branch 'hotel' into hotel-feature/guest-dashboard

Commit:
Removed hyperlink layout

Commit:
Formatted code

GaryMikoen committed:
added New Reservation button with link to page

GaryMikoen committed:
Added hotel link to homepage

Commit:
Added email field on button

Commit:
Merge branch 'hotel-feature/guest-dashboard' of https://github.com/mrjink/molveno into hotel-feature/guest-dashboard

Commit:
Added <p> for text on buttons

Commit:
Added molveno lake resort

Commit:
Added comma's to format the static text on the dashboard, added image background to hotel tile on hover

Mills315 committed:
middag 4/12/18
   bijna klaar

Mills315 committed:
Merge branch 'hotel' into hotel-feature/guest-reservation-room-options-screen

Mills315 committed:
aanpassing hotel/style.css
aangaande popovers

Commit:
Tijdelijke redirect van hotel pagina naar dashboard

Commit:
Added drill down details for rooms table

Commit:
Merge branch 'hotel' of https://github.com/mrjink/molveno into hotel

Commit:
Redirect tekst also in english gemaakt

Commit:
Merge branch 'hotel' of https://github.com/mrjink/molveno into hotel

Commit:
Merge branch 'hotel-feature/popup-room-info-screen' into hotel

Commit:
Hotel active on dashboard screen. I really need to create a branch for these small fixes...

Commit:
Merge branch 'hotel' of https://github.com/mrjink/molveno into hotel

Commit:
Made events text smaller on dashboard

Commit:
Interactive text on dashboard tiles will be smaller

Commit:
Added alert with emailaddress as a test

Mills315 committed:
Guest Reservation Room Options Screen klaar voor review

Mills315 committed:
Merge branch 'hotel' into hotel-feature/guest-reservation-room-options-screen

# Conflicts:
#	src/main/resources/static/css/hotel/style.css

Mills315 committed:
Guest Reservation Room Options Screen klaar voor review

GaryMikoen committed:
created roomservice for guest and also added styles for the invoice tables

GaryMikoen committed:
merged styles.css

Commit:
MySQL connection instead of H2

Commit:
Merge branch 'hotel' of https://github.com/mrjink/molveno into hotel

Commit:
Linked hotel tile to hotel page

Commit:
Merge branch 'hotel-feature/guest-dashboard' into hotel

Mills315 reviewed (approved):
Dit is zonder twijfel het werk van de mooie GITMASTER Lucas.

JustinStam reviewed (changes requested):
-

GaryMikoen reviewed (dismissed):
Goed gedaan beeldige gitmaster Lucas

Commit:
Changed properties

MoesKece committed:
Added GuestController, GuestRepository and minor fixes.

MoesKece committed:
Merge branch 'hotel-feature/receptionist-guests-info-backend' into hotel

Commit:
Button redirects to book room info screen

Commit:
Improved popup newsletter subscription

Commit:
Made dashboard semi-responsive

Commit:
Started with room-status-screem

Commit:
Finished most of the backend

Commit:
Merge branch 'hotel' of https://github.com/mrjink/molveno into hotel

Commit:
Added divs

GaryMikoen committed:
almost finished roomservice

GaryMikoen committed:
Merge branch 'hotel' into hotel-feature/guest-roomservice

Commit:
Fixed the reservation repo

Commit:
Merge branch 'master' into hotel

Commit:
Changed Integers to Longs in the models

Commit:
update

MoesKece committed:
Automagical redirect is removed, added three new pictures.

Mills315 committed:
Guest Reservation Room Options Screen klaar voor review

Commit:
Merge remote-tracking branch 'origin/hotel-feature/room-options-popup' into hotel-feature/room-status-screen

# Conflicts:
#	src/main/resources/static/css/hotel/style.css

Commit:
Merge remote-tracking branch 'origin/hotel-feature/hotel-frontpage' into hotel-feature/room-status-screen

MoesKece committed:
added pictures

Commit:
Merge remote-tracking branch 'origin/hotel-feature/hotel-frontpage' into hotel-feature/room-status-screen

Mills315 committed:
Guest Reservation Room Options Screen klaar
Room options popup klaar
Eerste javascript dingetjes geimplementeerd om niet statische informatie op de juiste plek te krijgen

Mills315 committed:
Guest Reservation Room Options Screen klaar
Room options popup klaar
Eerste javascript dingetjes geimplementeerd om niet statische informatie op de juiste plek te krijgen

Commit:
Merge remote-tracking branch 'origin/hotel-feature/room-options-popup' into hotel-feature/room-status-screen

Mills315 committed:
Merge branch 'hotel' into hotel-feature/room-options-popup

# Conflicts:
#	src/main/resources/application.properties
#	src/main/resources/static/css/hotel/style.css

Commit:
Link on dashboard to room info screen + changed booking popup to English (was partly Dutch)

Commit:
Made confirm button on booking popup redirect to guesthub. Added missing links to guesthub

Commit:
Fixed iFrame error by LW. Fixed text on dashboard

Commit:
Fixed iframes not working

Commit:
Merge branch 'hotel' of https://github.com/mrjink/molveno into hotel

Commit:
Merge branch 'hotel' into hotel-feature/room-status-screen

# Conflicts:
#	src/main/java/com/molvenolakeresort/services/impl/WebSecurityConfiguration.java
#	src/main/resources/application.properties

Mills315 committed:
Guest Reservation Room Options Screen klaar
Room options popup klaar
Eerste javascript dingetjes geimplementeerd om niet statische informatie op de juiste plek te krijgen

Commit:
Created containers for room status screen

Mills315 committed:
Merge remote-tracking branch 'origin/hotel' into hotel

# Conflicts:
#	src/main/resources/static/hotel/invoice-screen.html

GaryMikoen committed:
frontend layout roomservice finished, needs connection to backend data

GaryMikoen committed:
Merge branch 'hotel' into hotel-feature/guest-roomservice

Commit:
Frond end finished for room status screen. TODO: Connection to back end

Commit:
Merge branch 'hotel' into hotel-feature/room-status-screen

# Conflicts:
#	src/main/resources/static/css/hotel/style.css

Commit:
Fixed errorsssss

GaryMikoen committed:
added receptionist dashboard

GaryMikoen committed:
Merge branch 'hotel' into hotel-feature/receptionist-dashboard

Commit:
Added frontend to unsubscribe from newsletter

Commit:
Merge branch 'hotel' into hotel-feature/guest-dashboard-unsubscribe-email

Commit:
Fixed layout email tile

Commit:
Copy of room info screen

GaryMikoen committed:
changed format layout tax price container of roomservice

GaryMikoen committed:
added receptionist roomservice page

Commit:
changed name of webpage

Commit:
Merge branch 'hotel' into hotel-feature/guest-information

GaryMikoen committed:
added receptionist roomservice href

GaryMikoen committed:
Merge branch 'hotel' into hotel-feature/receptionist-dashboard

GaryMikoen committed:
added guest information page to receptionist dashboard

Commit:
Merge branch 'master' into hotel

Mills315 committed:
pdf generator dingen

Mills315 committed:
Merge branch 'hotel' into hotel-feature/pdf-generator

Commit:
Fixed the database filling

Commit:
Merge branch 'hotel' of https://github.com/mrjink/molveno into hotel
